doc/holochain_101/src/zome/define_zome.md

```diff
@@ -28,33 +28,33 @@ What are Rust macros? Generally speaking, they are code that will actually gener
 
 In the case of Zome development, it was discovered that much code could be saved from being written, by encapsulating it in a macro.
 
-That is how `define_zome!` came about. It is a Rust macro imported from the HDK which must be used for every Zome (unless you read the source code for it yourself and write something that behaves the same way!)
+That is how `#[zome]` came about. It is a Rust macro imported from the HDK which must be used for every Zome (unless you read the source code for it yourself and write something that behaves the same way!)
 
 The following is technically the most minimalistic Zome that could be implemented. It does nothing, but still conforms to the expectations Holochain has for a Zome.
 
 ```rust
 #[macro_use]
 extern crate hdk;
 
-define_zome! {
-    entries: []
-
-    init: || {
+#[zome]
+mod my_zome {
+    
+    #[init]
+    fn init() -> ZomeApiResult<()> {
         Ok(())
     }
 
-    validate_agent: |validation_data : EntryValidationData::<AgentId>| {
+    #[validate_agent]
+    fn validate_agent(validation_data : EntryValidationData::<AgentId>) -> ZomeApiResult {
         Ok(())
     }
-
-    functions: []
 }
 ```
 
-`entries` represents the validating entry type definitions. Note that it is an array, because there can be many. What validating entry types are will be [explained next](./entry_type_definitions.md).
+`init` tags the previously mentioned `init` callback that Holochain expects from every Zome. [Skip here for details.](./init.md)'
 
-`init` represents the previously mentioned `init` callback that Holochain expects from every Zome. [Skip here for details.](./init.md)
+`validate_agent` tags the callback that Holochain also expects from every Zome and is used to control who can join.
 
-`functions` is where the functions are defined. [Skip here for details.](./zome_functions.md)
+`functions` must be defined inside the `my_zome` module. [Skip here for details.](./zome_functions.md)
 
-These are the three *required* properties of `define_zome!`.
+`entries` must also be defined within the `my_zome` module. Entries are defined by annotating a function that returns a `ValidatingEntryType` with `#[entry_def]`. What validating entry types are will be [explained next](./entry_type_definitions.md).
```

Comment:
This statement could be interpreted as the name my_zome is required for the module. Is there another way to state this that provides the audience with a clear understanding that zome modules are allowed to have any name?